driver/utils/unicode_converter.h

```diff
@@ -0,0 +1,722 @@
+#pragma once
+
+#include "driver/platform/platform.h"
+#include "driver/utils/utils.h"
+
+#include <unicode/ustring.h>
+#include <unicode/ucnv.h>
+
+#include <string>
+#include <string_view>
+#include <vector>
+
+using ConverterPivotWideCharType = UChar;
+inline const std::string converter_pivot_wide_char_encoding = "UTF-16";
+
+template <typename CharType>
+inline auto make_raw_str(std::initializer_list<CharType> && list) {
```

Comment:
why is that `snake_case` while almost every other function is `lowerCamelCase` ?

Reply:
Yeah, that's intentional, kind of mimicking std convention here.